features/admin_views_auction.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Admin views public auction pages
+  As an administrator
+  I want to be able to click to edit an auction from the auction show page
+  So that I don't have to go back to the auctions admin panel
+
+  Scenario: Admin navigates to edit page from auction show page
+    Given I am an administrator
+    And I sign in
+    And there is an open auction
+    And I visit the auction page
+    Then I should see an "Edit Auction" link
+
+    When I click on the "Edit Auction" link
+    Then I should be on the admin form for that auction
+
+    When I click on the "Update" button
```

Comment:
ok so maybe this spec should be in a different file then? seems like are not quite testing `admin views auction` here